### DISCUSSION TITLE
how to set the maximum gpu memory occupied by jax project on a single GPU,

### DISCUSSION THREAD

**Discussion by andaolong** [jax-ml/jax#17639] [Q&A] [answered]
2023-9-18_11:03:20
I want to know how to set the maximum gpu memory occupied by jax project on a single GPU, so that I can use the GPU of our lab public server conveniently.

**Comment by anh-tong** [comment, marked as answer]
You can refer to the document here [GPU memory allocation](https://jax.readthedocs.io/en/latest/gpu_memory_allocation.html) and set environment variable as

`export XLA_PYTHON_CLIENT_MEM_FRACTION=.XX`

where `XX` is the percentage of GPU you want to use.

**Reply by andaolong** [reply]
Thank you very much for your advice. Your advice helped me solve the problem. Thank you again for your help.
